tests/unit_tests/test_wave_08.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+from swap_meet.item import Item
+from swap_meet.vendor import Vendor
+from swap_meet.clothing import Clothing
+from swap_meet.decor import Decor
+from swap_meet.electronics import Electronics
+
+# ~~~~~ display_inventory Tests ~~~~~
+def test_swap_similar_with_other_vendor():
+    # Arrange
+    # me
+    item_a = Decor(width=2, length=2)
+    item_b = Electronics(type="laptop")
+    item_c = Decor(width=4, length=3)
+    tai = Vendor(
+        inventory=[item_a, item_b, item_c]
+    )
+
+    # them
+    item_d = Clothing(fabric="Silk")
+    item_e = Decor(width=2, length=2)
+    item_f = Clothing(fabric="Cashmere")
+    jesse = Vendor(
+        inventory=[item_d, item_e, item_f]
+    )
+
+    # Act
+    result = tai.swap_similar_with_other_vendor(
+        other_vendor=jesse,
+        item=item_a
+    )
+    
+    assert result is True
+    assert len(jesse.inventory) == 3
+    assert len(tai.inventory) == 3
+    assert item_e in tai.inventory
+    assert item_b in tai.inventory
+    assert item_c in tai.inventory
+    assert item_d in jesse.inventory
+    assert item_a in jesse.inventory
+    assert item_f in jesse.inventory
+
+def test_swap_similar_with_other_vendor_not_found():
+    # Arrange
+    # me
+    item_a = Decor(width=2, length=2)
+    item_b = Electronics(type="laptop")
+    item_c = Decor(width=4, length=3)
+    tai = Vendor(
+        inventory=[item_a, item_b, item_c]
+    )
+
+    # them
+    item_d = Clothing(fabric="Silk")
+    item_e = Decor(width=2, length=2)
+    item_f = Clothing(fabric="Cashmere")
+    jesse = Vendor(
+        inventory=[item_d, item_e, item_f]
+    )
+
+    # Act
+    result = tai.swap_similar_with_other_vendor(
+        other_vendor=jesse,
+        item=item_b
+    )
+    
+    assert result is False
+    assert len(jesse.inventory) == 3
+    assert len(tai.inventory) == 3
+    assert item_a in tai.inventory
+    assert item_b in tai.inventory
+    assert item_c in tai.inventory
+    assert item_d in jesse.inventory
+    assert item_e in jesse.inventory
+    assert item_f in jesse.inventory
+
+def test_bad_instance_id():
```

Comment:
Love the added tests! 

For this one, I would consider adding some descriptors to the name. Since we're specifically looking at testing that we raise an error if an item subclass is instantiated with a bad id, we could use something like `test_create_item_subclass_invalid_id_raises_error` or `test_instantiation_raises_error_if_item_subclass_id_invalid`. Splitting this out into tests for each subclass could help reduce the length of the name since we could be more specific and could help us debug faster.